src/auto-evo/AutoEvoExploringTool.cs

```diff
@@ -2,17 +2,22 @@
 using System.Collections.Generic;
 using System.Globalization;
 using System.Linq;
+using AutoEvo;
 using Godot;
 
 /// <summary>
-///   Auto-evo exploring tool is a scene that contains numerous tool to debug, test, and explore auto-evo.
+///   Auto-evo exploring tool is a scene that contains numerous tools to debug, test, and explore auto-evo.
 ///   You can change the parameters to see how auto-evo responds, get a clearer view of how species evolves,
 ///   and start a game based on the simulation results.
+///   Partial class: GUI, auto-evo, world control
```

Comment:
This line should be in a remark section.

Reply:
Microbe.cs partial class is written like this. https://github.com/84634E1A607A/Thrive/blob/1e68bd326b91f5d8c217277479098d25ca11a4fa/src/microbe_stage/Microbe.cs#L9

Reply:
That's also not the best style in my eyes...  but here as the summary is already so long, putting the extra info outside the summary would be even better.